Added Preprocessing

MarchingTrombonist opened the pull request.

MarchingTrombonist committed:
Added Preprocessing

magser2000 committed:
Create union_dataset_years.py

Code to union the different year files of historical mortgage data together.

magser2000 commented:
Added some code to unionize the different files from the preprocessed pipeline so that all of our historical records are in one file to make it appropriate for me to use to make Tableau Dashboards for our use cases.

MarchingTrombonist committed:
New preprocessing functionality

deprecates union_dataset_years.py

concatenates files
drops unneeded cols
defines function and adds args for ease of use
